src/components/icons/DownloadInCircle.tsx

```diff
@@ -0,0 +1,43 @@
+import { SVGAttributes } from "react";
+
+const DownloadInCircle= (props: SVGAttributes<SVGElement>) => (
+  <svg
+    xmlns="http://www.w3.org/2000/svg"
+    width={24}
+    height={24}
+    viewBox="0 0 24 24"
+    fill="none"
+    {...props}
+  >
+    <path
+      stroke="#161517"
+      strokeWidth={1.5}
+      stroke-linecap="round"
+      stroke-linejoin="round"
```

Comment:
all the kebab-case props need to be camelCase (e.g. `strokeLinecap`, `strokeLinejoin`) otherwise they will trhow erros in the console